.github/workflows/build_test.yml

```diff
@@ -2,7 +2,7 @@ name: Build & Test
 
 on:
   push:
-  pull_request_target:
+  pull_request:
```

Comment:
this change is unrelated to this PR. I want to test if it improves how SonarCloud regonizes PRs

Reply:
This is causing the checks to fail
![image](https://user-images.githubusercontent.com/78716153/183584151-55df6022-df20-424c-9a62-e269dbd8bc0a.png)

Reply:
:/ I've undone this change